test/mockMuiStyles.js

```diff
@@ -14,9 +14,8 @@ const removeHash = (classes = {}) => {
 };
 
 jest.mock('@material-ui/styles/makeStyles', () => {
-  const makeStylesActual = require.requireActual(
-    '@material-ui/styles/makeStyles'
-  ).default;
+  const makeStylesActual = jest.requireActual('@material-ui/styles/makeStyles')
```

Comment:
`require.requireActual` is no longer a thing as of jestv26 and the recommended path is to use this jest version instead 👍